src/pybind/mgr/dashboard/frontend/src/locale/messages.cs.xlf

```diff
@@ -758,154 +944,155 @@
       </trans-unit>
       <trans-unit id="d2bcd3296d2850de762fb943060b7e086a893181" datatype="html">
         <source>Health</source>
-        <target>Zdraví</target>
+        <target>Health</target>
       </trans-unit>
       <trans-unit id="61e0f26d843eec0b33ff475e111b0c2f7a80b835" datatype="html">
         <source>Statistics</source>
-        <target>Statistiky</target>
+        <target>Statistics</target>
       </trans-unit>
       <trans-unit id="f8f74e5f683012b9c0702b1446011c6b9158bc67" datatype="html">
         <source>Please consult the 
           <x id="START_LINK" ctype="x-a" equiv-text="<a>"/>documentation
           <x id="CLOSE_LINK" ctype="x-a" equiv-text="</a>"/>
   on how to configure and enable the NFS Ganesha management functionality.
         </source>
-        <target>Ohledně nastavení a zapnutí funkce správy NFS Ganesha nahlédněte do 
-          <x id="START_LINK" ctype="x-a" equiv-text="<a>"/>dokumentace
+        <target>Please consult the 
+          <x id="START_LINK" ctype="x-a" equiv-text="<a>"/>documentation
           <x id="CLOSE_LINK" ctype="x-a" equiv-text="</a>"/>
+  on how to configure and enable the NFS Ganesha management functionality.
         </target>
       </trans-unit>
       <trans-unit id="7ffe39df9d88c972792bd8688b215392deb8313d" datatype="html">
         <source>Clients</source>
-        <target>Klienti</target>
+        <target>Clients</target>
       </trans-unit>
       <trans-unit id="f2dae0bda66f6a349444951c0379c28cda47d6d1" datatype="html">
         <source>Any client can access</source>
-        <target>Přistupovat může libovolný klient</target>
+        <target>Any client can access</target>
       </trans-unit>
       <trans-unit id="7882f2edb1d4139800b276b6b0bbf5ae0b2234ef" datatype="html">
         <source>Addresses</source>
-        <target>Adresy</target>
+        <target>Addresses</target>
       </trans-unit>
       <trans-unit id="9bd96fcf50863e685c74d0490392f46689ffbdb6" datatype="html">
         <source>Required field</source>
-        <target>Vyžadovaná kolonka</target>
+        <target>Required field</target>
       </trans-unit>
       <trans-unit id="a5f3f74c0f6925826cb2188576391c0da01a23f0" datatype="html">
         <source>Must contain one or more comma-separated values</source>
-        <target>Je třeba, aby obsahovalo jednu nebo více čárkou oddělovaných hodnot</target>
+        <target>Must contain one or more comma-separated values</target>
```

Comment:
I'm not sure if this sentence should really be replaced with English. Please correct me if I'm wrong, which is probably the case with my wild guess ;)